src/goose.rs

```diff
@@ -424,19 +425,6 @@ impl GooseTaskSet {
     }
 }
 
-/// Tracks the current run-mode of a client.
```

Comment:
I copied this from Locust -- but, it served no useful purpose so I'm removing it as part of this PR.